FTC SDK/FtcRobotController/src/main/java/com/qualcomm/ftcrobotcontroller/opmodes/OmniTeleOp.java

```diff
@@ -0,0 +1,79 @@
+package com.qualcomm.ftcrobotcontroller.opmodes;
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.Range;
+
+/**
+ * Created by Will Marchesi on 10/16/2015.
+ */
+public class OmniTeleOp extends OpMode{
+
+    DcMotor motorFrontLeft;
+    DcMotor motorFrontRight;
+    DcMotor motorBackLeft;
+    DcMotor motorBackRight;
+
+    public OmniTeleOp() {
+
+    }
+
+    @Override
+    public void init() {
+
+        motorFrontLeft = hardwareMap.dcMotor.get("motorFrontLeft");
+        motorFrontRight = hardwareMap.dcMotor.get("motorFrontRight");
+        motorBackLeft = hardwareMap.dcMotor.get("motorBackLeft");
+        motorBackRight = hardwareMap.dcMotor.get("motorBackRight");
+
+        telemetry.addData("Text", "*** Robot Data***");
+
+    }
+
+    @Override
+    public void loop() {
+
+        float driverLeftStick = -gamepad1.left_stick_y;
+        float driverRightStick = gamepad1.right_stick_x;
+
+        driverLeftStick = Range.clip(driverLeftStick, -1, 1);
+        driverRightStick = Range.clip(driverRightStick, -1, 1);
+
+        driverLeftStick = (float)scaleInput(driverLeftStick);
+        driverRightStick =  (float)scaleInput(driverRightStick);
+    }
+
+    @Override
+    public void stop() {
+
+    }
+
+    double scaleInput(double dVal)  {
+        double[] scaleArray = { 0.0, 0.05, 0.09, 0.10, 0.12, 0.15, 0.18, 0.24,
+                0.30, 0.36, 0.43, 0.50, 0.60, 0.72, 0.85, 1.00, 1.00 };
+
+        // get the corresponding index for the scaleInput array.
+        int index = (int) (dVal * 16.0);
+
+        // index should be positive.
+        if (index < 0) {
+            index = -index;
+        }
+
+        // index cannot exceed size of array minus 1.
+        if (index > 16) {
```

Comment:
What happens if someone changes the array size without updating this? Would be better to compare `index` to the current length of `scaleArray` - 1.